collector/benchmarks/bitmaps-3.1.0/.gitignore

```diff
@@ -0,0 +1,3 @@
+/target
+**/*.rs.bk
+#Cargo.lock
```

Comment:
I had to comment this line out because it was preventing `Cargo.lock` from being uploaded, which was preventing the tests from succeeding.